pkg/controller/performanceprofile/performanceprofile_controller_test.go

```diff
@@ -167,24 +160,14 @@ var _ = Describe("Controller", func() {
 
 			// verify KubeletConfig creation
 			kc := &mcov1.KubeletConfig{}
-			err = r.client.Get(context.TODO(), key, kc)
+			err := r.client.Get(context.TODO(), key, kc)
 			Expect(err).ToNot(HaveOccurred())
 		})
 
-		It("should do nothing on third reconcile loop", func() {
+		It("should create event on third reconcile loop", func() {
 			r := newFakeReconciler(profile)
 
-			result, err := r.Reconcile(request)
-			Expect(err).ToNot(HaveOccurred())
-			Expect(result).To(Equal(reconcile.Result{RequeueAfter: 10 * time.Second}))
```

Comment:
right here we're checking that the first reconcile loop results in a requeue, and that the others do not. How is that handled now with the new "reconcileTimes()" function?

Reply:
the requeue should always happen on the 1st reconcile loop, and that one is tested in line 112